### PR TITLE
Adds missing sound credit for #14169.

### PR DESCRIPTION
**Pull request by KingOfThePing**
I was oblivious, that CC0 sounds also need crediting, according to our contribution guidelines.

This PR adds the missing credit.

I am incapable of reading. One day, maybe, I will learn.

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!review

**Comment by KingOfThePing** [comment]
!bugfix

**Review by alsoandanswer — APPROVED** [review]

File: sound/attributions.txt [hunk 1]
```diff
@@ -1,4 +1,7 @@
 effects/creatures/vannatusk_attack.ogg - "Monster 2.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48673/
 effects/creatures/vannatusk_sound.ogg  - "Monster 3.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48674/
 effects/creatures/vannatusk_sound_2.ogg - "Monster 6.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48971/
-No changes were made to the sounds, and all credit goes to Sea Fury
\ No newline at end of file
+No changes were made to the sounds, and all credit goes to Sea Fury
+
+https://github.com/Aurorastation/Aurora.3/pull/14169 - All sounds used part of "Valhalla", licensed under CC0 1.0 Universal (CC0 1.0). Obtained from https://freesound.org/people/FilmmakersManual/
```
Comment (Arrow768) [inline sound/attributions.txt:6]: You need to specify the files added by your PR instead of linking the PR.

**Review by Arrow768 — CHANGES_REQUESTED** [review]
You need to use the established format in the file.
Alternatively the sound-files can be placed in a sounds/CC0/%author% folder with a readme.txt file in there containing the relevant information.

File: sound/attributions.txt (hunk 1 above)
Comment (KingOfThePing) [reply sound/attributions.txt:6]: Oh. Alright. I will check out the readme and fix this.

**Review by Alberyk — APPROVED** [review]

**Comment by KingOfThePing** [comment]
Closed, due to https://github.com/Aurorastation/Aurora.3/pull/14688 taking care of it. Thanks Gem.

**Pull request closed without merging**